src/exec_test.ts

```diff
@@ -69,7 +72,7 @@ describe('Exec', () => {
             when(fakeWebSocket.connect(`${path}?${args}`, null, anyFunction())).thenResolve(fakeConn);
 
             await exec.exec(
-                namespace, pod, container, cmd, osStream, errStream, isStream, false);
+                namespace, pod, container, cmd, osStream, errStream, isStream, nop, false);
```

Comment:
No new tests for the status handler being called?

Reply:
Added a test.